var/spack/repos/builtin/packages/r-devtools/package.py

```diff
@@ -13,27 +13,50 @@ class RDevtools(RPackage):
     version('1.11.1', sha256='51c876f9ddbfdf611f6ea0b06c0b46da8cefcb8cc98d60e06d576b61f0a06346')
 
     depends_on('r@3.0.2:', type=('build', 'run'))
     depends_on('r-httr@0.4:', type=('build', 'run'))
+    depends_on('r-httr@1.4.1:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-memoise@1.0.0:', type=('build', 'run'))
+    depends_on('r-memoise@1.1.0:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-whisker', when='@:1.9.9', type=('build', 'run'))
     depends_on('r-digest', type=('build', 'run'))
+    depends_on('r-digest@0.6.25', when='@3.2.0:' type=('build', 'run'))
     depends_on('r-rstudioapi@0.7.0:', type=('build', 'run'))
+    depends_on('r-rstudioapi@0.11.0:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-jsonlite', type=('build', 'run'))
+    depends_on('r-jsonlite@1.6.1:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-git2r@0.23.0:', type=('build', 'run'))
+    depends_on('r-git2r@0.26.1:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-withr', type=('build', 'run'))
+    depends_on('r-withr@2.1.2:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-usethis@1.5.0:', when='@2.0.0:', type=('build', 'run'))
+    depends_on('r-usethis@1.6.0:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-callr', when='@2.0.0:', type=('build', 'run'))
+    depends_on('r-callr@3.4.3:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-cli', when='@2.0.0:', type=('build', 'run'))
+    depends_on('r-cli@2.0.2:', when='@2.3.0:', type=('build', 'run'))
+    depends_on('r-covr@3.5.0:', when='@2.3.0:', type=('build', 'run'))
+    depends_on('r-crayon@1.3.4:', when='@2.3.0:', type=('build', 'run'))
+    depends_on('r-desc@1.2.0:', when='@2.3.0:', type=('build', 'run'))
+    depends_on('r-dt@0.13:', when='@2.3.0:', type=('build', 'run'))
+    depends_on('r-ellipsis@0.3.0:', when='@2.3.0:', type=('build', 'run'))
+    depends_on('r-glue@1.4.0:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-pkgbuild@1.0.3:', when='@2.0.0:', type=('build', 'run'))
+    depends_on('r-pkgbuild@1.0.6:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-pkgload@1.0.2:', when='@2.0.0:', type=('build', 'run'))
     depends_on('r-rcmdcheck@1.3.3:', when='@2.0.0:', type=('build', 'run'))
     depends_on('r-remotes@2.1.0:', when='@2.0.0:', type=('build', 'run'))
+    depends_on('r-remotes@2.1.1:', when='@2.3.0:', type=('build', 'run'))
+    depends_on('r-rlang@0.4.5:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-roxygen2@6.1.1:', when='@2.0.0:', type=('build', 'run'))
+    depends_on('r-roxygen2@7.1.0:', when='@2.3.0:', type=('build', 'run'))
     depends_on('r-sessioninfo@1.1.1:', when='@2.0.0:', type=('build', 'run'))
     depends_on('r-testthat@2.1.1:', when='@2.0.0:', type=('build', 'run'))
+    depends_on('r-testthat@2.3.2:', when='@2.3.0:', type=('build', 'run'))
+    depends_on('r-rversions@2.0.1:', when='@2.3.0:', type=('build', 'run'))
```

Comment:
Could you sort these dependencies in the same order they appear on https://cran.r-project.org/web/packages/devtools/index.html? That would make my life as a reviewer much easier 😄

Reply:
Sure, it's re-ordered ;)